[2024-03-21 01:29:08,777] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_dt_score 2024-03-21T01:28:00+00:00 [queued]>
[2024-03-21 01:29:08,803] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_dt_score 2024-03-21T01:28:00+00:00 [queued]>
[2024-03-21 01:29:08,803] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 01:29:08,803] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-03-21 01:29:08,804] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 01:29:08,830] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): compute_dt_score> on 2024-03-21T01:28:00+00:00
[2024-03-21 01:29:08,853] {standard_task_runner.py:52} INFO - Started process 14770 to run task
[2024-03-21 01:29:08,860] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'compute_dt_score', '2024-03-21T01:28:00+00:00', '--job-id', '8852', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmpo02oyeh7', '--error-file', '/tmp/tmpnj95l0zz']
[2024-03-21 01:29:08,860] {standard_task_runner.py:77} INFO - Job 8852: Subtask compute_dt_score
[2024-03-21 01:29:08,945] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.compute_dt_score 2024-03-21T01:28:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-21 01:29:09,083] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=compute_dt_score
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T01:28:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-21T01:28:00+00:00
[2024-03-21 01:29:09,603] {python.py:151} INFO - Done. Returned value was: None
[2024-03-21 01:29:09,620] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=compute_dt_score, execution_date=20240321T012800, start_date=20240321T012908, end_date=20240321T012909
[2024-03-21 01:29:09,675] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-21 01:29:09,733] {local_task_job.py:151} INFO - Task exited with return code 0
